integration/control_plane.go

```diff
@@ -7,14 +7,24 @@ import (
 
 // ControlPlane is a struct that knows how to start your test control plane.
 //
-// Right now, that means Etcd and your APIServer. This is likely to increase in
-// future.
+// A minimal control plane consists of an Etcd and an APIServer.
+//
+// Additionall control plane components can be added as `AdditionalComponents`.
```

Comment:
Drop one l, it's cleaner :-)

Reply:
Thanks. I will recheck on the docs and typos overall.